mlflow/utils/request_utils.py

```diff
@@ -24,6 +24,7 @@
         504,  # Gateway Timeout
     ]
 )
+DOWNLOAD_CHUNK_TIMEOUT_100M = 5 * 60
```

Comment:
can we make this an env var?

Reply:
I added one environment variable for this, but we shouldn't import it here because request_utils is designed to not import mlflow